bin/commands/init/stc/index.sh

```diff
@@ -112,7 +112,7 @@ else
   print_debug "- Copy ${prefix}.${ZWE_PRIVATE_DS_SZWESAMP}(ZWESLSTC) to ${tmpfile}"
   result=$(cat "//'${prefix}.${ZWE_PRIVATE_DS_SZWESAMP}(ZWESLSTC)'" | \
           sed "s/^\/\/STEPLIB .*\$/\/\/STEPLIB  DD   DSNAME=${authLoadlib},DISP=SHR/" | \
-          sed "s#^CONFIG=.*\$#CONFIG=${ZWE_CLI_PARAMETER_CONFIG}#" \
+          sed "s#^CONFIG=.*\$#CONFIG=$(convert_to_absolute_path ${ZWE_CLI_PARAMETER_CONFIG})#" \
```

Comment:
Change to conditional check:
- if it's already absolute path, do not convert.
- if it's relative path, or path starts with `~`, do the conversion but display a warning message like this:

```
CONFIG path defined in ZWESLSTC is converted into absolute path and may contain SYSNAME in the path. Please manually verify if this path works for you environment, especially when you are working in Sysplex environment.
```

Reply:
the convert_to_absolute_path function will return the absolute path as it is and wont prefix it with SYSTEM